core/Plugin/API.php

```diff
@@ -77,7 +77,7 @@ public static function getInstance()
     /**
      * Used in tests only
      * @ignore
-     * @deprecated
+     * @internal
```

Comment:
same here seems useful... however only for internal usage. deprecated flag was maybe set so it doesn't appear in the developer docs